lib/workers/repository/extract/index.ts

```diff
@@ -66,6 +67,10 @@ export async function extractAllDependencies(
       return { manager: managerConfig.manager, packageFiles };
     })
   );
+
+  // De-duplicate results using supercedesManagers
+  processSupercedesManagers(extractResults);
```

Comment:
Should we split this to a seperate PR?

Why not dedupe before extract? So we don't extract twice ans safe some time?

Reply:
You don't know until after extract if `bun` has `package.json` files which are also found in `npm`.

I think this function needs to be part of this PR so that you don't end up with the `package.json` being found by both `npm` and `bun`.